www-servers/sniproxy/files/sniproxy.init

```diff
@@ -0,0 +1,29 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+name="sniproxy daemon"
+description="Proxies incoming HTTP and TLS connections based on the hostname contained in the initial request."
+
+
+SNIPROXY_BINARY=${SNIPROXY_BINARY:-/usr/sbin/sniproxy}
+SNIPROXY_CONFFILE=${SNIPROXY_CONFFILE:-/etc/sniproxy/${SVCNAME}.conf}
+SNIPROXY_PIDFILE=${SNIPROXY_PIDFILE:-/var/run/sniproxy.pid}
+
+depend() {
+	need net
+	use logger
+}
+
+start() {
+	ebegin "Starting ${SVCNAME}"
+	start-stop-daemon --start --pidfile "${SNIPROXY_PIDFILE}" \
```

Comment:
This looks pretty standard. Please consider using the declarative form (using `command`, `command_args`, `pidfile` and so on, as explained in `man openrc-run`.